torch/export/serialization.py

```diff
@@ -0,0 +1,305 @@
+import torch
+from torch import Tensor
+from torch._subclasses.fake_tensor import FakeTensor
+from typing import Dict, List, Any
+import torch.export.export_schema as ex
+from torch.fx.interpreter import Interpreter
+import operator
+from itertools import chain
+
+should_export_buffer = False
+should_export_parameters_buffer = False
+should_export_node_meta = False
+
+def export_stroage(storage: torch.Storage) -> ex.Storage:
+    return ex.Storage(
+        data_location = ex.Storage.DataLocation.Internal,
+
+        # TODO: There should be a better way to get the data payload
+        # convert from list of bytes to byte string
+        data = ex.Buffer(
+            buffer = b''.join([i.to_bytes(1, 'big') for i in storage.untyped().tolist()]) \
```

Comment:
should have answered this dudes question lol https://discuss.pytorch.org/t/how-to-convert-a-pytorch-tensor-into-a-byte-string/96110

the real answer is something (maybe storage) should implement the buffer protocol and then just `bytes(blah)` would work

Reply:
if this doesn't exist already